### ISSUE TITLE
1.3.2 RC1 breaks apps build with webpack:webpack

### ISSUE THREAD

**Issue by laurentpayot** [meteor/meteor#6787]
1.3.2 RC1 breaks apps build with [webpack:webpack](https://github.com/thereactivestack/meteor-webpack/tree/master/packages/webpack/).
For instance:
```
git clone https://github.com/thereactivestack/kickstart-meteor-react.git
cd kickstart-meteor-react/
meteor update
meteor
npm install
meteor
```
works OK.

But if I do
```
meteor update --release 1.3.2-rc.1
meteor
```
I get
```
[[[[[ ~/meteor/debug/kickstart-meteor-react ]]]]]

=> Started proxy.                             
=> Started MongoDB.                           
webpack built e3e07b965296f735f4f8 in 1493ms /
Hash: e3e07b965296f735f4f8
Version: webpack 1.12.14
Time: 1493ms
 Asset    Size  Chunks       Chunk Names
web.js  222 kB       0       main
chunk    {0} web.js (main) 77.8 kB [rendered]
    [0] multi main 40 bytes {0} [built]
    [1] ./~/webpack-hot-middleware/client.js?path=http://localhost:3500/__webpack_hmr 4.18 kB {0} [built]
    [2] (webpack)/buildin/module.js 251 bytes {0} [built]
    [3] ./~/querystring/index.js 127 bytes {0} [built]
    [4] ./~/querystring/decode.js 2.4 kB {0} [built]
    [5] ./~/querystring/encode.js 2.09 kB {0} [built]
    [6] ./~/strip-ansi/index.js 161 bytes {0} [built]
    [7] ./~/ansi-regex/index.js 135 bytes {0} [built]
    [8] ./~/webpack-hot-middleware/client-overlay.js 1.73 kB {0} [built]
    [9] ./~/ansi-html/index.js 4.02 kB {0} [built]
   [10] ./~/html-entities/index.js 231 bytes {0} [built]
   [11] ./~/html-entities/lib/xml-entities.js 2.98 kB {0} [built]
   [12] ./~/html-entities/lib/html4-entities.js 6.57 kB {0} [built]
   [13] ./~/html-entities/lib/html5-entities.js 49 kB {0} [built]
   [14] ./~/webpack-hot-middleware/process-update.js 3.88 kB {0} [built]
   [15] ./src/client.js 41 bytes {0} [built] [1 error]

ERROR in ./src/client.js
Module not found: Error: Cannot resolve module 'TodoApp/client' in /home/laurent/meteor/debug/kickstart-meteor-react/src
 @ ./src/client.js 3:0-25
=> Errors prevented startup:                  
   
   While processing files with webpack:webpack (for target os.linux.x86_64):
   src/server.js: ./src/server.js
   Module not found: Error: Cannot resolve module 'TodoApp/server' in /home/laurent/meteor/debug/kickstart-meteor-react/src
   resolve module TodoApp/server in /home/laurent/meteor/debug/kickstart-meteor-react/src
   looking for modules in /home/laurent/meteor/debug/kickstart-meteor-react/node_modules
   /home/laurent/meteor/debug/kickstart-meteor-react/node_modules/TodoApp doesn't exist (module as directory)
   [/home/laurent/meteor/debug/kickstart-meteor-react/node_modules/TodoApp]
   @ ./src/server.js 3:0-25
   
=> Your application has errors. Waiting for file change.
```

**Comment by benjamn** [comment]
Good catch! Check out the commit above if you're curious what was going on here. This will be fixed in `1.3.2-rc.3`.

**Comment by laurentpayot** [comment]
Looks like it's working now :+1:
